setup.py

```diff
@@ -55,16 +55,17 @@
     long_description=long_description,
     long_description_content_type='text/markdown',
     packages=[
-        'arm_characterization',
-        'drive_characterization',
-        'elevator_characterization',
-        'logger_gui',
-        'cli',
-        'utils',
-        'newproject',
-        'robot'
+        'frc_characterization',
+        'frc_characterization.arm_characterization',
+        'frc_characterization.drive_characterization',
+        'frc_characterization.elevator_characterization',
+        'frc_characterization.cli',
+        'frc_characterization.logger_gui',
+        'frc_characterization.newproject',
+        'frc_characterization.robot',
+        'frc_characterization.utils'
     ],
```

Comment:
You should be able to replace this with a call to find_packages.

Reply:
I've heard that it is more robust to include them manually - is this not true?

Reply:
It probably is, but you would have to remember to keep the list up-to-date.

Reply:
I think that's worth it; it's not that much of a maintenance hassle.

Reply:
For robotpy we always use find_packages() just so that we don't accidentally forget something when publishing a new package. I'm always using packages installed in development mode, so it's really easy to forget to update the list.